automol/form/_form.py

```diff
@@ -193,15 +173,13 @@ def string(fml, hyd=True):
     return fml_str
 
 
-def string2(fml):
+def string2(fml: dict[str:int]) -> str:
```

Comment:
Minor syntax correction: `dict[str, int]`.

Having `string()` and `string2()` in here really gets under my skin, but let's leave it for now. Eventually, I would like to combine this with `string()` and simply add a flag `show_ones: bool=False` to the former to toggle showing the 1s on/off.